theme/n21/views/help/lern-nuggets.hbs

```diff
@@ -7,6 +7,6 @@
 	{{/content}}
 
 	{{#content "page"}}
-        Diese Seite ist leider nicht verfügbar.
+        {{$t "theme.brb.help.text.unfortunatelyThisPageIsNotAvailable"}}
```

Comment:
is this on purpose?  brb message in n21 theme?

Reply:
okk. means there should be a general place for those texts which use in several theme?